packages/databricks-vscode/src/configuration/auth/MetadataService.ts

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import * as http from "node:http";
+import * as crypto from "node:crypto";
+import {Disposable} from "vscode";
+import {AddressInfo} from "node:net";
+import {ApiClient} from "@databricks/databricks-sdk";
+
+export class MetadataService implements Disposable {
```

Comment:
Do we expect all databricks processes within the same project folder to be able to access the metadata service even if the process is launched from outside vscode? This might require treewalking in all other sdks then.

Reply:
The Metadata service itself won't solve that because the URL is not stable. We'll need your `env` solution to achieve that.